pkg/controller/test/context_builder.go

```diff
@@ -40,6 +41,7 @@ import (
 	cmfake "github.com/jetstack/cert-manager/pkg/client/clientset/versioned/fake"
 	informers "github.com/jetstack/cert-manager/pkg/client/informers/externalversions"
 	"github.com/jetstack/cert-manager/pkg/controller"
+	fakediscovery "github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/validation/plugins/fake"
```

Comment:
We should move this package to a better shared location for the project. `/test/unit/discovery` seems right.